pkg/drivers/kic/oci/oci.go

```diff
@@ -571,3 +576,38 @@ func ShutDown(ociBin string, name string) error {
 	glog.Infof("Successfully shutdown container %s", name)
 	return nil
 }
+
+//containerLogs will print out the logs for a container
+func containerLogs(ociBin string, name string) (*RunResult, error) {
+	if ociBin == Docker {
+		return runCmd(exec.Command(ociBin, "logs", "--timestamps", "--details", name))
+	}
+	// podman doesn't have --details
+	return runCmd(exec.Command(ociBin, "logs", "--timestamps", name))
+}
+
+//PrintPostPortem will print relevant docker/podman infos after a container fails
```

Comment:
This seems to log instead of print (normally associated to stdout). Perhaps `LogContainerStatus`?

Reply:
how about  LogContainerDebug?